importer/preserve.py

```diff
@@ -0,0 +1,69 @@
+from cms.blogs.models import Blog, BlogIndexPage
+from cms.pages.models import Page
+from django.core.management.base import BaseCommand
+
+
+def preserve(obj):
+    """Save updates to obj (including first-write) without
+    trashing the publication/revision dates"""
+    fpa = obj.first_published_at
```

Comment:
It's generally good practice to avoid initialisms in variable names, just for clarity.

Reply:
That's fair. I was finding the obj was getting lost in
```
first_published_at = obj.first_published_at
obj.first_published_at = first_published_at
```